recipe/install-gdb.sh

```diff
@@ -25,8 +25,22 @@ pushd ${PREFIX}
       *script*executable*)
       ;;
       *executable*)
-        ${SRC_DIR}/gcc_built/bin/${CHOST}-strip --strip-all -v "${_file}"
+        ${SRC_DIR}/gcc_built/bin/${CHOST}-strip --strip-all -v "${_file}" || :
       ;;
     esac
   done
 popd
+
+if [[ "${ctng_cpu_arch}" == "x86_64" ]]; then
+  old_vendor="-conda_cos6-linux-gnu-"
+else
+  old_vendor="-conda_cos7-linux-gnu-"
+fi
+
+for exe in `ls ${PREFIX}/bin/*-conda-linux-gnu-*`; do
+  nm=`basename ${exe}`
+  new_nm=${nm/"-conda-linux-gnu-"/${old_vendor}}
+  if [ ! -f ${PREFIX}/bin/${new_nm} ]; then
+    ln -s ${exe} ${PREFIX}/bin/${new_nm}
+  fi
+done
```

Comment:
Can you move the above few lines to a separate script and source it here?

Reply:
Right. That seems a lot better. :)